pkg/pulumiyaml/run_plugin_version_test.go

```diff
@@ -29,11 +28,11 @@ resources:
 	assert.False(t, diags.HasErrors())
 
 	got := plugins
-	want := autogold.Want("test-plugins", []Plugin{{
```

Comment:
The `autogold` librarry doesn't work with bazel.  I don't feel like its usage is crucial anywhere we have it, the couple of places we're using it are barely worth having the extra dependency, especially because this shouldn't change frequently anyway.

Reply:
:'(

I love snapshot testing. Why does this not work?

Reply:
For the same reason this change is needed: https://github.com/pulumi/pulumi-yaml/pull/568#discussion_r1567173412

Bazel creates a separate directory structure with symlinks to the original files to run the tests.  So `autogold` can't copy the file back to the repo.  Note that we can still work the whole `PULUMI_ACCEPT` thing work via just running the tests via the normal Go tooling, just the `autogold` library wants to doesn't like the symlinks even when it's not supposed to change anything for some reason.